src/QueryStats.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace YdbPlatform\Ydb;
+
+class QueryStats
+{
+
+    private $query_phases;
+    protected $compilation = null;
+    protected $process_cpu_time_us = 0;
+    protected $query_plan = '';
+    protected $query_ast = '';
+    protected $total_duration_us = 0;
+    protected $total_cpu_time_us = 0;
+    public function __construct(\Ydb\TableStats\QueryStats $queryStats)
+    {
+        $this->compilation = $queryStats->getCompilation();
+        $this->query_ast = $queryStats->getQueryAst();
+        $this->query_plan = $queryStats->getQueryPlan();
+        $this->query_phases = $queryStats->getQueryPhases();
+        $this->total_cpu_time_us = $queryStats->getTotalCpuTimeUs();
+        $this->total_duration_us = $queryStats->getTotalDurationUs();
+        $this->process_cpu_time_us = $queryStats->getProcessCpuTimeUs();
+    }
+
+    /**
+     * @return \Google\Protobuf\Internal\RepeatedField
+     */
+    public function getQueryPhases(): \Google\Protobuf\Internal\RepeatedField
```

Comment:
Need to wrap all protobuf messages.

SDK must not have any proto message in public interface.